app/controllers/steps/duty_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Steps
   class DutyController < BaseController
+    helper_method :title
+
     def show
       @duty_options = duty_options
       @gbp_to_eur_exchange_rate = Api::MonetaryExchangeRate.for('GBP').exchange_rate.round(4) unless user_session.import_into_gb?
```

Comment:
this means that if `user_session.import_into_gb?` is nil,
@gbp_to_eur_exchange_rate will remain nil, right?
maybe a ternary operator would be clearer

Reply:
Yeah, this is the guard clause style of basically doing:

```ruby
unless user_session.import_into_gb?
  @gbp_to_eur_exchange_rate = Api::MonetaryExchangeRate.for('GBP').exchange_rate.round(4)
end
```

I've not actually touched that here, though

Reply:
oh, you didn't! sorry
... your code is clearer anyway